packages/block-editor/src/utils/transform-styles/index.js

```diff
@@ -23,7 +18,7 @@ import wrap from './transforms/wrap';
  * @return {Array} converted rules.
  */
 const transformStyles = ( styles, wrapperClassName = '' ) => {
-	return map( styles, ( { css, baseURL } ) => {
+	return Object.values( styles ?? [] ).map( ( { css, baseURL } ) => {
```

Comment:
Can be nullish, which is why the nullish coalescing:

https://github.com/WordPress/gutenberg/blob/5e2850dbdd28a09f153989b6308f258cbd37cdfa/packages/block-library/src/html/preview.js#L25

And we're doing the `Object.values()` because there are instances where we pass an object:

https://github.com/WordPress/gutenberg/blob/7973d9cc5890b64c78e86cb86e1d52d8df55fc69/packages/block-editor/src/hooks/font-size.js#L273

although most of the rest of the usages pass an array of styles. `Object.values()` will correctly yield us the values for arrays, so we're good with that part, as well.